src/signals/types.nim

```diff
@@ -87,3 +92,15 @@ type WhisperFilterSignal* = ref object of Signal
 
 type DiscoverySummarySignal* = ref object of Signal
   enodes*: seq[string]
+
+
+
+proc findById*(self: seq[Chat], id: string): int =
```

Comment:
Nitpick: rather call this `findIndexById`, as the current name implies the return value to be a `Chat` type